### ISSUE TITLE
Smartparens and NXML

### ISSUE THREAD

**Issue by pronobis** [Fuco1/smartparens#397]
When closing the tags in nxml-mode, smartparens will conflict with the nxml tag completion and result in double closing >> for a tag. Example:

```
<tag>
|
```

then

```
<tag>
<|>
```

then

```
<tag>
</tag>|>
```

**Comment by pronobis** [comment]
Ok, it seems that I'm getting closer to the problem. It's the nxml-mode autocomplete function (https://www.gnu.org/software/emacs/manual/html_node/nxml-mode/Inserting-end_002dtags.html) that adds /tag> not company. When setting `nxml-slash-auto-complete-flag` to `nil`, I don't get the autocompletion of the ending tags and in principle the problem goes away. Still, it would be nice to get the nxml autocompletion of ending tags work with smartparens. 

Basically, to reproduce:
1) Create a new xml file test.xml
2) Switch to nxml-mode
3) Enable auto-completion of end tags `(setq nxml-slash-auto-complete-flag t)`
3) Type `<tag>`
4) Type `</`
Observe that you get `</tag>>`

**Comment by albcorp** [comment]
A workaround is to set `(setq nxml-slash-auto-complete-flag nil)`. In this case, you must do an additional keystroke for each closing tag. Namely, at step 5 above, type the slash, and at new step 6,  do `completion-at-point` which is commonly bound to `C-M-i` aka `M-tab`.

**Comment by Fuco1** [comment]
Easier solution might be to add a hook to smartparens that deletes the closing `>` when you enter `/` right after `<`

    (sp-local-pair 'nxml-mode "<" ">" :post-handlers '(("[d1]" "/")))
